Assignment4/rubykoom-21C008/task06.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1snaoJ7Cpt9-waIihN43xosnVFXx4I2nU
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't work in Python, as it's notebook magic

Reply:
Line deleted in both .py docs